automatic/simplewall/simplewall.nuspec

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>simplewall</id>
+    <version>2.3.4</version>
+    <authors>Henry++</authors>
+    <summary>Simple tool to configure Windows Filtering Platform.</summary>
+    <description><![CDATA[Simple tool to configure Windows Filtering Platform (WFP) which can configure network activity on your computer.
+
+The lightweight application is less than a megabyte, and it is compatible with Windows Vista and higher operating systems.
+
+## Features
+
+* Simple interface without annoying pop ups
+* Rules editor (create your own rules)
+* Internal blocklist rules (block Windows spy / telemetry)
+* Dropped packets information with notification and logging to a file feature (win7+)
+* Allowed packets information with logging to a file feature (win8+)
+* Windows Subsystem for Linux (WSL) support (win10)
+* Windows Store support (win8+)
+* Windows services support
+* Free and open source
+* Localization support
+* IPv6 support
+
+## Notes
+
+* For correct working, need administrator rights.
+]]></description>
+    <title>simplewall</title>
+    <owners>chocolatey</owners>
+    <tags>admin foss simplewall wfp firewall</tags>
+    <projectUrl>https://www.henrypp.org/product/simplewall</projectUrl>
+    <projectSourceUrl>https://github.com/henrypp/simplewall</projectSourceUrl>
+    <licenseUrl>https://github.com/henrypp/simplewall/blob/master/LICENSE</licenseUrl>
+    <iconUrl>https://cdn.jsdelivr.net/gh/chocolatey/chocolatey-coreteampackages@71a47d708fd1ac175d0fc5184dc5b01624e2afda/icons/simplewall.png</iconUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages</packageSourceUrl>
+    <releaseNotes>https://github.com/henrypp/simplewall/blob/master/CHANGELOG.md</releaseNotes>
+    <dependencies>
+      <dependency id="simplewall.install" version="[2.3.4]" />
+    </dependencies>
+    </metadata>
+</package>
```

Comment:
you'll need to add an empty `<files />` element, otherwise both the Readme.md and update.ps1 file would be included here.